masterlist.yaml

```
@@ -9866,33 +9866,157 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/20371/'
         name: 'Less Annoying Magic Experience'
-  - name: 'bgMagic(D|E)V.esp'
+  - name: 'bgMagic[DE]V\.esp'
     after: [ 'Tiefling Races.esp' ]
+    req: [ 'DLCShiveringIsles.esp' ]
     inc: [ 'Tiefling Races.esp' ]
-    # doNotClean
-    tag: [ SpellStats ]
+    tag:
+      - EffectStats
+      - Graphics
+      - Sound
+      - SpellStats
+  - name: 'bgMagicDV.esp'
+    dirty:
+      - <<: *quickClean
+        crc: 0x6E266800
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 3
+  - name: 'bgMagicEV.esp'
+    dirty:
+      - <<: *quickClean
+        crc: 0x03BFDD76
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 3
   # LAME - Optional modules #
-  - name: 'bgMagic(D|E)VAddEnVar.esp'
-    req: [ 'DLCShiveringIsles.esp' ]
-  - name: 'bgMagic(Alchemy|Lightningbolt|PotionNumberSoulGemValue|ShaderLifeDetect)\.esp'
-    after:
-      - 'bgMagicDV.esp'
-      - 'bgMagicEV.esp'
   - name: 'bgMagicDV(Bonus|ItemSigil)\.esp'
     req: [ 'bgMagicDV.esp' ]
   - name: 'bgMagic(Bonus|ItemSigil)\.esp'
     req: [ 'bgMagicEV.esp' ]
-  - name: 'bgMagic(DV)?ItemSigil.esp'
-    # doNotClean
+  - name: 'bgMagic(Alchemy|Lightningbolt|PotionNumberSoulGemValue|ShaderLifeDetect)\.esp'
+    req:
+      - name: 'bgMagicEV.esp'
+        condition: 'not file("bgMagicDV.esp")'
```

You'd still need a load after rule for the DV version.

```suggestion
    req:
      - name: 'bgMagicEV.esp'
        condition: 'not file("bgMagicDV.esp")'
    after: [ 'bgMagicDV.esp' ]
```